src/content/language/patterns.md

```diff
@@ -390,7 +390,7 @@ if (data is Map<String, Object?> &&
     var name = user[0] as String;
     var age = user[1] as int;
     print('User $name is $age years old.');
-  }
+  } 
```

Comment:
This added space seems like an unnecessary change to an otherwise unchanged file.